backend/local/socket.ts

```diff
@@ -19,25 +21,29 @@ export const startWebSocketServer = () => {
     const socketId = generateId(16);
     (ws as WebsocketWithId).id = socketId;
 
-    const config = {
+    const config: ConfigWithHandler = {
       connectionId: socketId,
       tableName: 'scrum-poker-local',
       ddb,
       handler: {
-        postToConnection: ({ ConnectionId, Data }: { ConnectionId: string; Data: unknown }) => {
+        postToConnection: ({ ConnectionId, Data }) => {
+          // $metadata is required in the return type
+          if (!Data) return Promise.resolve({ $metadata: {} });
           const client = [...wss.clients].find(
-            (client) => ConnectionId === (client as WebsocketWithId).id
+            (client) => ConnectionId === (client as WebsocketWithId).id,
           );
-          const resultPromise = client
-            ? new Promise<void>((resolve, reject) =>
-                client.send(Data, (error) => (error ? reject(error) : resolve()))
+          return client
+            ? new Promise((resolve, reject) =>
+                client.send(
+                  Data as Exclude<typeof Data, Readable | Blob | ReadableStream<unknown>>,
```

Comment:
I am not sure were this type incompatibility now comes from, but I would be fine with this cast as it is only need for running the app locally.

Reply:
👍